models/AppState.py

```diff
@@ -1,5 +1,77 @@
 """Application state class"""
 
+import sys
+from numpy import array as np_array, min as np_min, ptp as np_ptp
+from models.PyCryptoBot import PyCryptoBot
+from models.TradingAccount import TradingAccount
+from models.exchange.binance import AuthAPI as BAuthAPI
+from models.exchange.coinbase_pro import AuthAPI as CAuthAPI
+
+def binanceMinimumOrderBase(app:PyCryptoBot, account:TradingAccount):
```

Comment:
It would be better having all this functions as member of AppState class.

Reply:
I tried that but had issues calling the functions from the class method. It didn't like calling the function with self./cls.

It seems to work moving them out like that. I will try again this evening. It was how I had it originally but didn't seem to work or at least pylint was complaining.